README.md

```diff
@@ -186,10 +186,10 @@ Minikube tools to be installed and available on your PATH.
 
    * `vs-kubernetes` - Parent for Kubernetes-related extension settings
        * `vs-kubernetes.namespace` - The namespace to use for all commands
-       * `vs-kubernetes.kubectl-path` - File path to the kubectl binary. Note this is the binary file itself, not just the directory containing the file. On Windows, this must contain the `.exe` extension.
+       * `vscode-kubernetes.kubectl-path` - File path to the kubectl binary. Note this is the binary file itself, not just the directory containing the file. On Windows, this must contain the `.exe` extension.
```

Comment:
💡Wondering and please add your thoughts: I wonder if this is deprecated: and the new setting exist above `vs-kubernetes: {...}` scope. So this should be removed all together.